CMakeLists.txt

```diff
@@ -253,7 +253,10 @@ FILE(GLOB_RECURSE SOURCE "src/*.cc" "src/*.h")
 FILE(GLOB_RECURSE CUDA "src/*.cu" "src/*.cuh")
 
 # add nnvm to source
-FILE(GLOB_RECURSE NNVMSOURCE "nnvm/src/*.cc" "nnvm/src/*.h" "nnvm/include/*.h")
+FILE(GLOB_RECURSE NNVMSOURCE
+  nnvm/src/c_api/*.cc
```

Comment:
Looks ok. Can you keep the headers in?  This helps IDE's such as CLion know to index the file.

Reply:
updated